h5py/h5d.pyx

```diff
@@ -151,6 +151,63 @@ def open(ObjectID loc not None, char* name, PropID dapl=None):
     """
     return DatasetID(H5Dopen(loc.id, name, pdefault(dapl)))
 
+IF HDF5_VERSION >= (1, 14, 0):
+    @with_phil
+    def rw_multi(list dataset_ids, list mspace_ids, list fspace_ids,
```

Comment:
This could be broken up into `read_multi` and `write_multi` for readability, but they would be identical except for one parameter passed to `dset_rw_multi`.